containers/test/net.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+sudo docker network create -d sriov --subnet=172.31.80.0/20 -o netdevice=eth0 mynet
```

Comment:
The subnet should depend on your local network - maybe we can make it a parameter? Wrapping network initialization is definitely a good idea!

Reply:
Another idea: add a second parameter for two types of networking - `sriov` and `host`. This way we can easily support a simpler networking mode, at least for testing.

Reply:
That is a great idea. I was going to parameterize this.